man/systemd-notify.xml

```diff
@@ -174,6 +177,20 @@
         this option set is prone to race conditions in all other cases.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--exec</option></term>
+
+        <listitem><para>If specified <command>systemd-notify</command> will execute another command line
+        after it completed its operation, replacing its own process. If used, the list of assignments to
+        include in the message sent must be followed by a <literal>;</literal> character (as separate
+        argument), followed by the command line to execute. This permits "chaining" of commands, i.e. issuing
+        one operation, followed immediately by another, without changing PIDs.</para>
+
+        <para>Note that many shells interpret <literal>;</literal> as their own separator for command lines,
```

Comment:
bikeshed: wouldn't ` -- ` allow to avoid the escaping issue? I think other tools use that as a separator

Reply:
nah, "--" is special to getopt() itself, and we cannot turn that off even if we wanted. it means "do no interpret paramaters after this as switches". So we really should stay away from that, it has an established, different meaning already.